api/controllers/createPatron.js

```diff
@@ -166,11 +166,11 @@ function createPatron(req, res) {
             req,
             res,
             statusCode,
-            modelResponse.errorResponseData(responseObject),
+            modelResponse.errorResponseData(responseObject) // eslint-disable-line comma-dangle
           );
         } else {
           renderResponse(req, res, 500, modelResponse.errorResponseData(
-            collectErrorResponseData(null, '', '', '', ''),
+            collectErrorResponseData(null, '', '', '', '') // eslint-disable-line comma-dangle
```

Comment:
Do you need all these comments here?

Reply:
@EdwinGuzman the comments help the linter ignore the errors (ie missing comma).  I think that when we can get the error down to 0 errors, it's easier for us as developers to notice when there is a different error.

I'll chat about it with you today to see if there's a better way.

With the commas in, Node v6 has a problem.
With the commas out, the linter has a problem.
So I've removed the commas and added comments for the linter.